### PR TITLE
feat: add service template

### PR DESCRIPTION
**Pull request by Tanya-atatakai**
## Proposed changes

Add service template for Modius
Based on:
https://github.com/valory-xyz/modius-quickstart/blob/main/run_service.py#L434
https://github.com/valory-xyz/modius-quickstart/blob/main/run_service.py#L899

## Types of changes

What types of changes does your code introduce?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -8,4 +8,5 @@ DEV_RPC=
 GNOSIS_RPC=
 ETHEREUM_RPC=
 OPTIMISM_RPC=
-BASE_RPC=
\ No newline at end of file
+BASE_RPC=
+MODE_RPC=
```
Comment (truemiller) [inline .env.example:12]: do we have mode rpc gateway yet @Tanya-atatakai?

File: frontend/constants/serviceTemplates.ts [hunk 2]
```diff
@@ -256,6 +256,112 @@ export const SERVICE_TEMPLATES: ServiceTemplate[] = [
...
+        use_staking: true,
+        cost_of_bond: 20000000000000000000,
+        monthly_gas_estimate: 5260000000000000,
+        fund_requirements: {
+          agent: 5000000000000000,
+          safe: 0,
+        },
+      },
+    },
+    env_variables: {
```
Comment (jmoreira-valory) [inline frontend/constants/serviceTemplates.ts:284]: Missing RPC variables  (computed)

File: frontend/constants/serviceTemplates.ts [hunk 3]
```diff
@@ -256,6 +256,112 @@ export const SERVICE_TEMPLATES: ServiceTemplate[] = [
...
+        description: '',
+        value: '',
+        provision_type: EnvProvisionType.COMPUTED,
+      },
+      TENDERLY_PROJECT_SLUG: {
+        name: 'Tenderly project slug',
+        description: '',
+        value: '',
+        provision_type: EnvProvisionType.COMPUTED,
+      },
```
Comment (jmoreira-valory) [inline frontend/constants/serviceTemplates.ts:308]: These variables are not 'computed', these are user-input variables.
Please check the remaining ones.

File: frontend/constants/serviceTemplates.ts [hunk 4]
```diff
@@ -256,6 +256,112 @@ export const SERVICE_TEMPLATES: ServiceTemplate[] = [
...
+        name: 'Initial assets',
+        description: '',
+        value: '',
+        provision_type: EnvProvisionType.COMPUTED,
+      },
+      SELECTED_STRATEGIES: {
+        name: 'Selected strategies',
+        description: '',
+        value: '',
+        provision_type: EnvProvisionType.COMPUTED,
```
Comment (jmoreira-valory) [inline frontend/constants/serviceTemplates.ts:361]: Same, this can possibly be a fixed value.

Comment (Tanya-atatakai) [reply frontend/constants/serviceTemplates.ts]: have updated as I understood it (using this https://github.com/valory-xyz/modius-quickstart/blob/main/run_service.py#L403)

File: .env.example (hunk 1 above)
Comment (Tanya-atatakai) [reply .env.example:12]: afaik we don't and should use the public one

**Pull request merged**